### PR TITLE
add initial legal and housing resource pages

### PR DESCRIPTION
**Pull request by levis501**

### CODE REVIEW

File: client/src/index.css [hunk 1]
```diff
@@ -3,3 +3,7 @@ body {
   padding: 0;
   font-family: sans-serif;
 }
+
+.ui.label {
+  margin: 0.1em;
```
Comment (levis501) [inline client/src/index.css]: Make the vertical spacing between labels non-zero. When there are more than a couple Labels in a Table.Cell, they show up on multiple lines. The default zero vertical margin made them smoosh together.

Comment (turtles) [reply client/src/index.css]: Making it a global style complicates the code down the line. It's not clear that .ui.label is overridden in the main stylesheet. It's better to make it explicit, and also only affect labels in table cells, so there are less sources of error. To do this, make it a CSS class and set the className prop on the Label to that CSS class.

`.tableCellLabel {
margin: 0.1em;
}`

`<Label className='tableCellLabel'> ...`

Comment (levis501) [reply client/src/index.css]: Oh, that makes total sense.  will do

**Comment by levis501** [comment]
I have some bug where the css class not getting picked up by the Label.   Will look into it.

**Comment by turtles** [comment]
It's probably CSS specificity rules. ui.label is more specific than a custom class, so it's chosen. Try using `important!`. https://stackoverflow.com/questions/47129141/react-semantic-ui-and-a-custom-css-class

**Comment by levis501** [comment]
@turtles I think this is good to go.  I've fully specified the CSS class for the label and that seems to do the trick.  I'll need you to perform merge, however, because of permissions.

**Pull request merged**